templates/_bundle_builder/Dockerfile

```diff
@@ -19,28 +19,28 @@ RUN apt-get update && \
 #
 
 RUN apt-get update && \
-    \
-    DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --yes libass-dev libfaac-dev libmp3lame-dev libopus-dev libsdl2-dev libtheora-dev libvorbis-dev libvpx-dev libx264-dev libx265-dev yasm
-
-RUN curl -L -o /build/ffmpeg.tar.gz https://ffmpeg.org/releases/ffmpeg-7.1.tar.gz && \
-    \
-    cd /build && \
-    \
-    tar -xvzf ffmpeg.tar.gz && \
-    \
-    cd ffmpeg-* && \
-    \
-    mkdir build && \
-    \
-    cd build && \
-    \
-    ../configure --enable-gpl --enable-libass --enable-libfreetype --enable-libmp3lame --enable-libopus --enable-libtheora --enable-libvorbis --enable-libvpx --enable-libx264 --enable-libx265 --enable-nonfree --enable-sdl2 --enable-shared --enable-version3 && \
-    \
-    make -j$(nproc) && \
-    \
-    make install && \
-    \
-    rm -rf /build/*
+	\
+	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --yes libass-dev libfaac-dev libmp3lame-dev libopus-dev libsdl2-dev libtheora-dev libvorbis-dev libvpx-dev libx264-dev libx265-dev yasm
+
+RUN curl --location --output /build/ffmpeg.tar.gz https://ffmpeg.org/releases/ffmpeg-7.1.tar.gz && \
```

Comment:
@Akos-Kreutz, don't we need to create the folder `build`? Is it already being created by some other code?

Reply:
It is created on line 7.
https://github.com/liferay-release/liferay-docker/pull/145/files#diff-2756bc306808244b14c224b0886ee73c84c83f63a957c1b3f6055f5b622e9185R7